OxfordBodleian/Juel-Jensen/BDLaethf34.xml

```diff
@@ -150,38 +627,198 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc type="MagicFormula">. The text is written in a poor hand in dark brownish ink.</desc>  
+                                    <locus target="#1r"/>
+                                    <q xml:lang="gez">ክርክምና፡ ክምና፡ ኵልኩልና፡ ቀርቃና፡ ሜሎስ፡ <del rend="effaced" unit="chars" quantity="1"/>ዴን፡ ቆቁና፡ ፌወቆቁሬሁ፡ ቁሽ፡ 
+                                        ቃለ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ዘተሰወረኒ፡ ክሰት፡ ሊተ፡ አባ፡</q>
+                                    </item>
+                                <item xml:id="a2">
+                                    <desc>Title of the book and name of the owner.
+                                        The text was written in black ink by <persName ref="PRS13276Thyne" role="owner"><roleName type="rank">Captain</roleName> 
+                                        W.K. Thyne</persName> on <date>4 March 1864</date>, in <placeName ref="LOC0127IHA">Aden</placeName>. The text also mentions the previous owner of the manuscript, 
+                                        <persName ref="PRS8943Speedy" role="owner">Tristram Speedy</persName>.
+                                    </desc>  
+                                    <locus target="#1v"/>
+                                    <q xml:lang="en">Ethiopic Mazmoor or Abyssinian Prayer Book. <persName ref="PRS13276Thyne" role="owner">J.W.K. Thyne</persName> 
+                                        from his sincere friend <persName ref="PRS8943Speedy" role="owner">T.C.S.S</persName>. Aden 4.3.64</q>
+                                </item>
+                                <item xml:id="a3">
+                                    <desc type="Condemnation">. The text is crudely written in black ink and encircled in a frame. It is mostly erased. However, the name of 
+                                        <persName ref="PRS13275HaylaGi"><roleName type="title">ʾAbbā</roleName> Ḫayla Giyorgis</persName>, possibly a later owner of the 
+                                        manuscript, is still legible.</desc>  
+                                    <locus target="#3ra"/>
+                                    <q xml:lang="gez">ዘሰአታተ፡ ዘአባ። ኃይለ፡ ጊ<del rend="erasure">ዮርጊስ፡</del> <del rend="erasure" unit="chars" quantity="4"/> ወዘፈሀቆ፡ በስልጣነ፡ 
+                                        ጴጥሮስ፡ ወጳውሎስ፡ ዉጉዝ፡ ለይ<del rend="erasure">ኩን፡</del> </q>
+                                </item>
+                                <item xml:id="a4">
+                                    <locus target="#84v"/>
+                                    <desc type="ProtectivePrayer">. The text is written in a crude and untrained hand in the upper margin of the folio.</desc>  
+                                    <q xml:lang="gez">ማርያም፡ ማርዲኒስ፡ ኅቱስ፡ ናዳስ፡ ዘሰማዕኩ፡ ከመ፡ አጽንዕ፡ ወዘአጽና<supplied reason="omitted" resp="MV">ዕ</supplied>ኩ፡
+                                    ከመ፡ ኢይግድፍ፡ ክሥት፡ ሊተ፡ ለገብርከ፡ እገሌ፡ ግበር፡ ዘ፫፯፡ ዘቢብ፡ ፯፡ ጊዜ፡ ድግም፡ ዘእብነት፡ ዘደብተራ፡ ጌታሁን፡
```

Comment:
ዘእብነት፡ ዘደብተራ፡ ጌታሁን፡
I think this is an interesting note that might refer to a "note book" of a dabtara. We may keep it in mind for our introductions. We could cross check it with Gidena (or with his thesis).

Reply:
Any cross reference to contributions on "magic" material is the most welcome!

Reply:
Gidena's thesis "Ethiopian Abǝnnät Manuscripts: Organizational Structure, Language Use, and Orality" you find here: 
https://ediss.sub.uni-hamburg.de/handle/ediss/7027

Reply:
I've added a file called "Introduction" in the folder to collect such remarks